demo/demo.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#Test oc tool
+#Assumes User is logged in to cluster
+
+path=$1
+url=$2
+
+# Will create spring rest deployment
+cd $path/basic-nginx
```

Comment:
this needs to change to another `pushd... popd`. otherwise this will break the commit logic below.